paket.dependencies

```diff
@@ -1,18 +1,13 @@
-source https://nuget.org/api/v2
+source https://api.nuget.org/v3/index.json
 redirects: on
-framework net461
+framework net461, netstandard1.6, netcoreapp1.1
 
 github logary/logary src/Logary.Facade/Facade.fs
 
 nuget Argu
 nuget FsCheck
-nuget FSharp.Core
 nuget BenchmarkDotNet
-nuget BenchmarkDotNet.Toolchains.Roslyn
-nuget BenchmarkDotNet.Core
 nuget NuGet.CommandLine
 nuget Mono.Cecil >= 0.10-beta5
-
-group build
```

Comment:
When I saw that FAKE was not used, I removed it, but I decided to use it as the build system. It will be in its separate group in the future.